test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -164,8 +165,13 @@ def run_ec2_tensorflow_inference(image_uri, ec2_connection, grpc_port, region, t
             )
         if telemetry_mode:
             check_telemetry(ec2_connection, container_name)
+    except:
+        inference_test_failed = True
+        remote_out = ec2_connection.run(f"docker logs {container_name}")
```

Comment:
Should we use `hide=True` and `warn=True` over here?

Reply:
Updated in latest commit